wiki/Beatmap_Editor_Guides/Compressing_Videos/en.md

```diff
@@ -1,3 +1,6 @@
+Compressing Videos
```

Comment:
hmm, I guess I'll look into this article and might do a rewrite later (probably in the next month or so)